java/openmldb-import/pom.xml

```diff
@@ -62,9 +64,8 @@
         <!-- https://mvnrepository.com/artifact/org.apache.curator/apache-curator -->
         <dependency>
             <groupId>org.apache.curator</groupId>
-            <artifactId>apache-curator</artifactId>
-            <version>5.1.0</version>
-            <type>pom</type>
+            <artifactId>curator-client</artifactId>
```

Comment:
many module has deps to curator, it is better to define curator version in a single place( the parent pom) later. 

Should we exclude zookeeper as explained in https://curator.apache.org/zk-compatibility-34.html ?

Reply:
openmldb-import is not a submodule now. Maybe later?